Error: Cannot find module '../../options'

```
[webpack-cli] Error: Cannot find module '../../options'
Require stack:
- /Users/dh/Sites/wn/node_modules/@pmmmwh/react-refresh-webpack-plugin/lib/utils/normalizeOptions.js
- /Users/dh/Sites/wn/node_modules/@pmmmwh/react-refresh-webpack-plugin/lib/utils/index.js
- /Users/dh/Sites/wn/node_modules/@pmmmwh/react-refresh-webpack-plugin/lib/index.js
```

@pmmmwh/react-refresh-webpack-plugin: 0.5.0-beta.2
webpack: 5.33.2
react-refresh: 0.10.0
laravel-mix: 6.0.16

An error occurred after upgrading the version from 0.5.0-beta.1 to 0.5.0-beta.2

--- a/package.json
+++ b/package.json
@@ -31,6 +31,7 @@
     "client",
     "lib",
     "loader",
+    "options",
     "overlay",
     "sockets",
     "types"

